templates/layouts/aw-install-simg.hbs

```diff
@@ -15,58 +15,85 @@
       <p>Before installing AsteroidOS to your watch, make sure you are aware of the capabilities and limitations of AsteroidOS on the {{title}}. The following table should summarize the current support of this watch:</p>
       {{> body }}
       <h1>Preparation</h1>
-      <div class="install-preparation-box">
-          <img src="{{assets}}/img/{{deviceName}}.jpg" class="install-preparation-img"><br>
-      </div>
       <div class="install-preparation-box">
         <h3>Download AsteroidOS 1.0</h3>
         <a class="btn btn-primary" href="https://release.asteroidos.org/1.0/{{deviceName}}/asteroid-image-{{deviceName}}.ext4" role="button">asteroid-image-{{deviceName}}.ext4</a>
         <a class="btn btn-primary" href="https://release.asteroidos.org/1.0/{{deviceName}}/zImage-dtb-{{deviceName}}.fastboot" role="button">zImage-dtb-{{deviceName}}.fastboot</a>
-        <h3>Install Fastboot & Adb</h3>
+        <br><br><i>Place the files in your users "Downloads" folder for the later commands to work</i><br>
+        <h3>Install Fastboot & ADB</h3>
+        <b>Linux systems</b> with apt package manager
+        <br><br>
         <pre><code class="bash">sudo apt install android-tools-adb android-tools-fastboot android-tools-fsutils</code></pre>
-        <p><i>Note: previous command may differ depending on your OS</i></p>
+        <br>
+        <b>Windows users</b> download this installer <a href="https://forum.xda-developers.com/showthread.php?t=2317790">published on XDA by shimp208</a>
+        <br><br>
+        <a class="btn btn-primary" href="https://androidfilehost.com/?fid=746010030569952951" role="button">Minimal ADB and Fastboot 1.43</a>
```

Comment:
That androidfilehost experience is also quite bad. If we could distribute tools we have compiled and tested ourselves, on our own server that would be much better.

Reply:
Ok, i extracted fastboot and adb myself from android sdk.
We could simply distribute it as zip but i have no idea about licensing.
a windows driver with git repo would be available here as i saw: https://adb.clockworkmod.com/

Going to make a series of test if flashing is successfull